superset-frontend/plugins/legacy-preset-chart-deckgl/package.json

```diff
@@ -32,12 +32,10 @@
     "deck.gl": "8.5.2",
     "jquery": "^3.4.1",
     "lodash": "^4.17.15",
-    "mapbox-gl": "^2.4.0",
```

Comment:
from root `node_module` get this module

Reply:
Echoing @villebro's comment, we may need this to be a peer dependency so that the plugin continues to work in contexts outside of Superset.

Reply:
the `mapbox-gl` come from `react-map-gl`, so we didn't need to add this.